Controllers/AuthenticationController.cs

```diff
@@ -2,48 +2,56 @@
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using static Communify_Backend.Models.AuthenticationModels;
+using static Communify_Backend.Models.TokenModels;
 
-namespace Communify_Backend.Controllers
+namespace Communify_Backend.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AuthenticationController : Controller
 {
-    [ApiController]
-    [Route("api/[controller]")]
-    public class AuthenticationController : Controller
+    private readonly IAuthenticationService authenticationService;
+
+    public AuthenticationController(IAuthenticationService authenticationService)
```

Comment:
Belki tek satir parametreli constructor yapabiliriz gosterdigim gibi bu sadece bir fikir.

Reply:
İlerde gerekecek herhangi bir fonksiyon icin ekstra servis ekleme durumları için böyle kalması daha iyi olmaz mi?